style.css

```diff
@@ -0,0 +1,127 @@
+body{
+    font-weight: 400;
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 18px;
+    font-size: 12px;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+    background-color: #eceff1;
+    color: #4C4C4C;
+}
+h1{
+    font-size: 24px;
+    font-weight: lighter;
+}
+p{
+    font-size: 14px;
+}
+a{
+    color: #0077C5;
+    text-decoration: none;
+}
+nav ul li{
+    display: inline;
+}
+.upper-nav{
+    height: 60px;
+    width: 100%;
+    margin-bottom: 30px;
+    display: inline-flexbox;
+    margin-top: 10px
+}
+.sign-in-btn{
+   float: right;
+}
+.sign-in-btn p{
+    margin: 0;
+    font-size: 12px;
+    padding-right: 10px;
+}
+.sign-in-btn a{
+    text-decoration: none;
+    color: #0077C5;
+}
+
+fieldset{
+    width: 30%;
+    margin: 0 auto;
+    background-color: #ffffff;
+    border: 1px solid #c7c7c7;
+}
+.form-content p{
+    text-align: left;
+    padding-left: 13px;
+    
+}
+.main-content{
+    margin-top: 20px;
+}
+input{
+    height: 34px;
+    padding: 0 8px;
+    width: 320px;
+    border: 1px solid #C1C5C8;
+    border-radius: 3px;
+}
+.mint-button{
+    margin-top: 30px;
+    width: 93%;
+    color: #ffffff;
+    background-color: #0077C5;
+    border: #0077C5;
+    border-radius: 4px;
+    margin-bottom: 20px;
+    height: 44px;
+}
+.mint-mini-button{
+    padding: 0 20px;
+    border-radius: 4px;
+    border: 1px solid #4C4C4C;
+    height: 32px;
+    color: #ffffff;
+    background-color: #0077C5;
+}
+footer{
+    text-align: center;
+    margin-top: 40px;
+}
+footer p{
+    color: #505050;
+    font-size: 12px;
+    margin: 10px 0;
+}
+footer a{
+    color: #505050;
+    text-decoration: none;
+}
+
+label{
+    font-size: 14px;
+    float: left;
+    padding-left: 12px;
+    padding-top: 10px;
+}
+#standard-call{
```

Comment:
Also, remember its better to use classes instead of ids.

Reply:
Thank You
We have learnt a lot from your reviews and we are sure to do better with our next projects.

Reply:
You are welcome don't forget to merge to the master also.